python/tchannel/messages/common.py

```diff
@@ -85,6 +87,10 @@ def compute_checksum(checksum_type, args, csum=0):
     # TODO figure out farm32 cross platform issue
     elif checksum_type == ChecksumType.farm32:
         raise NotImplementedError()
+    elif checksum_type == ChecksumType.crc32c:
+        cfun = crcmod.predefined.mkCrcFun('crc-32c')
```

Comment:
could this call into crcmod be done once at module time? or is cfun a stateful callable?